src/mod/codecs/mod_opus/mod_opus.c

```diff
@@ -104,6 +105,10 @@ static opus_codec_settings_t default_codec_settings_16k = {
 	/*.samplerate*/ 0
 };
 
+/*! \brief Storage for data about frame content VAD, etc. */
+opus_packet_info_t opus_packet_info;
+rtp_codec_data_t rtp_codec_data;
```

Comment:
It does not look like these two declarations should be here. 
You can rename the current struct codec_control_state to codec_control_state_encode , and create a new struct with the two named codec_control_state_decode .